qcodes_contrib_drivers/drivers/Keysight/Keysight_E36313A.py

```diff
@@ -0,0 +1,111 @@
+from qcodes import VisaInstrument
+from qcodes import Instrument
+from qcodes.instrument.channel import InstrumentChannel
+
+
+class E36313AChannel(InstrumentChannel):
+    """
+
+    """
+    def __init__(self, parent: Instrument, name: str, chan: int) -> None:
+        """
+        Args:
+            parent (Instrument): The instrument to which the channel is
+            attached.
+            name (str): The name of the channel
+            channum (int): The number of the channel in question (1-2)
+        """
+        # Sanity Check inputs
+        if name not in ['ch1', 'ch2', 'ch3']:
+            raise ValueError("Invalid Channel: {}, expected 'ch1' or 'ch2' or 'ch3'"
+                             .format(name))
+        if chan not in [1, 2, 3]:
+            raise ValueError("Invalid Channel: {}, expected '1' or '2' or '3'"
+                             .format(chan))
+
+        super().__init__(parent, name)
+
+        self.add_parameter('source_voltage',
+                           label="Channel {} Voltage".format(chan),
+                           get_cmd='SOURCE{:d}:VOLT?'.format(chan),
+                           get_parser=float,
+                           set_cmd='SOURCE{:d}:VOLT {{:.8G}}'.format(chan),
+                           unit='V')
+
+        self.add_parameter('source_current',
+                           label="Channel {} Current".format(chan),
+                           get_cmd='SOURCE{:d}:CURR?'.format(chan),
+                           get_parser=float,
+                           set_cmd='SOURCE{:d}:CURR {{:.8G}}'.format(chan),
+                           unit='A')
+
+        self.add_parameter('voltage',
+                           get_cmd='MEAS:VOLT? (@{:d})'.format(chan),
+                           get_parser=float,
+                           label='Channel {} Voltage'.format(chan),
+                           unit='V')
+
+        self.add_parameter('current',
+                           get_cmd='MEAS:CURR? (@{:d})'.format(chan),
+                           get_parser=float,
+                           label='Channel {} Current'.format(chan),
+                           unit='A')
+
+        self.add_parameter('resistance',
+                           get_cmd='MEAS:RES? (@{:d})'.format(chan),
+                           get_parser=float,
+                           label='Channel {} Resistance'.format(chan),
+                           unit='ohm')
```

Comment:
I do not see anything in the manual which say it can measure resistance?  I am looking at this https://literature.cdn.keysight.com/litweb/pdf/E36311-90008.pdf